pkg/skaffold/docker/auth.go

```diff
@@ -27,13 +27,14 @@ import (
 	"github.com/docker/cli/cli/config/configfile"
 	"github.com/docker/distribution/reference"
 	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/registry"
 	"github.com/moby/moby/pkg/homedir"
 	"github.com/pkg/errors"
-	"github.com/sirupsen/logrus"
 )
 
 const (
-	configFileDir = ".docker"
+	configFileDir    = ".docker"
+	dockerHubAuthKey = "ttps://index.docker.io/v1/"
```

Comment:
missing a `h` in the link `ttps://index.docker.io/v1/` ?

Reply:
:-)